wallet/reference/non-evm-apis/starknet-snap-api.md

```diff
@@ -0,0 +1,868 @@
+---
+description: See the Starknet Snap API reference.
+sidebar_position: 1
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Starknet Snap API
+
+When connected to the [Starknet Snap](../../how-to/use-non-evm-networks/starknet/index.md), dapps
+can use the Starknet Snap API to interact with users' Starknet accounts (for example, to send transactions).
+
+Currently, the [`get-starknet`](https://github.com/starknet-io/get-starknet) library only supports the
```

Comment:
do we really need to specific what SNAP api does get-starknet implemented

as the starknet community (builders) usually following the general standard on client side , which is get-starknet / starknet.js, they have no idea what Starknet Snap API is

i think we can change the angle, instead of telling ppl what API that we supported in get-starknet, we can tell ppl MetaMask Starknet Snap is almost fully support the functionality of get-starknet v3.3.0 (except on/off event listener)

Reply:
@Montoya we have a discussion, what i try to say is using an API to represent a starknet network operation/function is not correct
e.g get-starknet does not integrate starkNet_estimateFee, but it doesnt mean it cant do it, it is still possible to estimate the fee

so me and @khanti42 come up an conclusion that we use operation/method as the category, and each entity (get-starknet, snap), we attach the way to how to perform such action

otherwise, we make the get-starknet only support very few snap api, and seem we only use snap api to interact with the network

Reply:
Ah great, that makes sense